### DISCUSSION TITLE
Create Swagger Blocks

### DISCUSSION THREAD

**Discussion by gautamsinghania95** [fastapi/fastapi#8002] [Questions] [answered]
Currently, in the Docs URL, I can only see one block with all APIs like this:
<img width="1400" alt="Screenshot 2019-11-12 at 11 16 11 AM" src="https://user-images.githubusercontent.com/15277899/68645380-dc933400-053d-11ea-8a2c-06842240eb95.png">

Do I have the control of the name, and can I create multiple such blocks?

**Comment by euri10** [comment, marked as answer]
You can use tags

**Comment by gautamsinghania95** [comment]
Thanks for your help. Closing the issue.

**Comment by tiangolo** [comment]
Thanks for the help here @euri10 ! :clap: :bow: 

Thanks for reporting back and closing the issue @gautamsinghania95  :+1:
